scormxblock/scormxblock.py

```diff
@@ -506,6 +510,26 @@ def _publish_progress(self, completion):
         """
         self.runtime.publish(self, 'completion', {'completion': completion})
 
+    def calc_progress_measure(self, scorm_data):
```

Comment:
we can use calculate or compute(avoid using abbreviations)

Reply:
Sure